frontend/src/components/DietPlan/StandardMealItem.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+import NativeIcon from "../Icon/NativeIcon";
+import { Colors } from "@/constants/Colors";
+import { MaterialCommunityIconsNames } from "@/types/iconTypes";
+
+interface StandardMealItemProps {
+  quantity: number;
+  iconName: MaterialCommunityIconsNames;
```

Comment:
Just pass an the icon. You are locking yourself to just the material Community   icons. If you ever want to use something else you will have to change all the typing instead of just changing the icon itself being passed.